src/v/cluster/members_backend.cc

```diff
@@ -411,6 +464,7 @@ void members_backend::calculate_reallocations_after_recommissioned(
 
         meta.partition_reallocations.push_back(std::move(reallocation));
     }
+    co_return;
```

Comment:
Was it a conscious choice to not enforce this new limit in recommissioning case because we lose the guarantees of reinstating the entire previous replica set if we return early? If so are we still prone to this movement storm while recommissioning nodes?  or may be I misread the code, just want to double check.

Reply:
Recommissioning only does interrupt currently ongoing movements. Since the number of ongoing movements is limited then there is no risk that there will be to many concurrent moves at once.